### PR TITLE
Added tags for Vue templates

### PR DESCRIPTION
**Pull request by armanio123**
Fixes Vue templates not appearing on the "New Project Dialog".

### CODE REVIEW

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptVuejsApp/TypeScriptVuejsApp.vstemplate [hunk 1]
```diff
@@ -12,6 +12,9 @@
     <NumberOfParentCategoriesToRollUp>1</NumberOfParentCategoriesToRollUp>
     <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
     <TemplateGroupID>Node.js</TemplateGroupID>
+    <LanguageTag>Typescript</LanguageTag>
+    <PlatformTag>Windows</PlatformTag>
```
Comment (amcasey) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptVuejsApp/TypeScriptVuejsApp.vstemplate]: Why Windows?  Seems like this would work anywhere?

Comment (armanio123) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptVuejsApp/TypeScriptVuejsApp.vstemplate]: I agree but there's no option for 'any' on the templates and the others are misleading. Here's the documentation [Built-in tags](https://docs.microsoft.com/en-us/visualstudio/ide/template-tags?view=vs-2019#built-in-tags). The other templates are tagged as Windows as well.

Comment (amcasey) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptVuejsApp/TypeScriptVuejsApp.vstemplate]: The example in the docs makes it look like you can tag multiple platforms.

Comment (armanio123) [reply Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptVuejsApp/TypeScriptVuejsApp.vstemplate]: I suppose I can add Linux and MacOS as well as per Node.js support. I'll fix it in second.

**Pull request merged**